tests/dbshell/test_mysql.py

```diff
@@ -112,3 +118,17 @@ def get_command_line_arguments(self, connection_settings, parameters=None):
         if parameters is None:
             parameters = []
         return DatabaseClient.settings_to_cmd_args(connection_settings, parameters)
+
+    def test_charset_configured(self):
+        self.assertEqual(
+            ['mysql', '--user=someuser', '--password=somepassword',
+             '--host=somehost', '--port=444', 'somedbname', '--default-character-set=utf8'],
+            self.get_command_line_arguments({
+                'NAME': 'somedbname',
+                'USER': 'someuser',
+                'PASSWORD': 'somepassword',
+                'HOST': 'somehost',
+                'PORT': 444,
+                'OPTIONS': {},
+                'DEFAULT-CHARACTER-SET': 'utf8',
```

Comment:
We should add `charset` to `OPTIONS`:
```suggestion
                'OPTIONS': {
                    'charset': 'utf8',
                },
```